src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/Transaction.java

```diff
@@ -0,0 +1,72 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.Optional;
+
+public class Transaction extends Section {
+    @Getter(lazy = true)
+    private final ReferenceTransactionNumber referenceTransactionNumber =
```

Comment:
Are all of these segments needed right now? I don't think we know the contents of the message yet

Reply:
Maybe not all the segments, but `Transaction` is needed as `EdifactParser` uses it in its helper methods

Reply:
I'll go through them now and check to see which ones are needed/unneeded

Reply:
Removed the unnecessary segments 😄